### ISSUE TITLE
pusht task success rate is low after trained 300k in original arguments

### ISSUE THREAD

**Issue by alpttex19** [huggingface/lerobot#3221]
### Ticket Type

❓ Technical Question

### Environment & System Info

```Shell
main branch, linux 22.04, python 3.12, torch 2.10.0
```

### Description

lerobot-train --output_dir=outputs/train/diffusion_pusht_lerobot_main --policy.type=diffusion --env.type=pusht --dataset.repo_id=lerobot/pusht  --seed=100000 --batch_size=64 --steps=400000 --policy.push_to_hub=false  --eval_freq=25000  --save_freq=25000  --wandb.enable=true

<img width="1382" height="643" alt="Image" src="https://github.com/user-attachments/assets/6291f3a2-c95d-4279-919a-764d162031e8" />

### Relevant logs or stack trace

```Shell

```

### Checklist

- [ ] I have searched existing tickets to ensure this isn't a duplicate.
- [x] I am using the latest version of the `main` branch.
- [ ] I have verified this is not an environment-specific problem.

**Comment by jashshah999** [comment]
Before investigating hyperparameters, rule out dataset issues: `lerobot-doctor check lerobot/pusht --checks statistics,actions,training` verifies statistics match expected ranges and checks for anomalous episodes. Eliminates data quality from the equation.

https://github.com/jashshah999/lerobot-doctor
